iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -506,31 +507,19 @@ TEST_F(PortManager_test, DestroyEventVariableAndAddNewOneSucceeds)
     std::vector<iox::popo::EventVariableData*> eventVariableContainer;
 
     // first acquire all possible event variables
-    for (unsigned int i = 0U; i < iox::MAX_NUMBER_OF_EVENT_VARIABLES; i++)
-    {
-        auto eventVariableDataResult = m_portManager->acquireEventVariableData(process);
-        EXPECT_THAT(eventVariableDataResult.has_error(), Eq(false));
-        eventVariableContainer.push_back(eventVariableDataResult.value());
-    }
-
-    // so now no event variables should be available
-    auto eventVariableDataResult = m_portManager->acquireEventVariableData(process);
-    EXPECT_THAT(eventVariableDataResult.has_error(), Eq(true));
+    acquireMaxNumberOfEventVariables(
+        process, [&](auto eventVariableDataResult) { eventVariableContainer.push_back(eventVariableDataResult); });
```

Comment:
nitpicking, but this is not a result but an actual event variable

Reply:
renamed